quarto-render/README.md

```diff
@@ -0,0 +1,103 @@
+# A Github Action to Render Quarto Documents
+
+This repository stores the `quarto-render` [GitHub Action](https://github.com/features/actions) for use with [Quarto](https://quarto.org/) projects. This action will call `quarto render` on the root directory of the repo. It also has example workflows for some common uses centered around web publishing.
+
+## Examples
+
+### Rendering a static `quarto` site and pushing to `gh-pages` branch
+
+If your quarto project has `freeze: true`, you can use this Github action to render your documents and publish your workflow to Github pages by using something like the following workflow:
+
+```yaml
+# .github/workflows/render-and-publish.yaml
+name: Render and deploy Quarto files
+on:
+  push:
+  pull_request:
+
+jobs:
+  quarto-render-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Install Quarto and render"
+        uses: pommevilla/quarto-render@main
```

Comment:
We should probably change the path of the action here
```suggestion
        uses: quarto-dev/quarto-actions/quarto-render@v1
```

And I'll move the v1 tag once merged ?

Reply:
🤔 I used `install-quarto` and not `quarto-install`... maybe that was not a good choice ? 

Should we try to be close and use `render-quarto` ? I understand why `quarto-render` though as this is close to the command line call. 

Just sharing thoughts. I don't mind keeping this as is really

Reply:
Ooh, I didn't think of that! I'll go ahead and rename this to `render-quarto` to match yours.